nx-dev/ui-blog/src/lib/blog-details.tsx

```diff
@@ -64,16 +65,28 @@ export function BlogDetails({ post }: BlogDetailsProps) {
             {post.title}
           </h1>
         </header>
-        {post.cover_image && (
-          <div className="mx-auto mb-16 aspect-[1.7] w-full max-w-screen-md">
-            <Image
-              className="h-full w-full object-cover md:rounded-md"
-              src={post.cover_image}
-              alt={post.title}
-              width={1400}
-              height={735}
+        {post.podcastYoutubeId && post.podcastSpotifyId ? (
+          <div className="mx-auto mb-16 w-full max-w-screen-md">
+            <EpisodePlayer
```

Comment:
can we adjust how the episode player visualizes

![Screenshot 2024-08-08 at 18 26 43](https://github.com/user-attachments/assets/b185e2dd-8b36-48c3-8188-39ccd01250d0)

like make it much more compressed
- moving the icons to the left below the embed and much smaller
- moving the "switch" button and "podcasts" button to the very right below the embed and make them links rather than buttons